.travis.yml

```diff
@@ -0,0 +1,32 @@
+language: node_js
+
+sudo: false
+
+node_js:
+- 10
+- 12
+
+before_install:
+- echo -e "machine github.ibm.com\n  login $GITHUB_OAUTH_TOKEN" > ~/.netrc
+# -  'openssl aes-256-cbc -K $my_key -iv $my_iv -in myservice.env.enc -out myservice.env -d || true'
+
+script:
+- npm run build
+- npm run test-travis || travis_terminate 1
+- npm run lint
+- npm run check-packages
```

Comment:
Probably better to have this in reverse order? That way it can fail fast.

```
 - npm run check-packages
 - npm run lint
 - npm run test-travis || travis_terminate 1
 - npm run build
```

Reply:
this is our typical "recipe" for building a node SDK

Reply:
There's no harm in putting "check-packages" and "lint" first, but "build" must occur before the tests